src/content/language/libraries.md

```diff
@@ -89,29 +89,24 @@ import 'package:lib1/lib1.dart' show foo;
 import 'package:lib2/lib2.dart' hide foo;
 ```
 
-<a id="deferred-loading"></a>
```

Comment:
Can you keep this extra anchor fragment target? It's been used for a while and there are a few links from other sites to it.